ldap2pg.master.yml

```diff
@@ -55,17 +55,36 @@ postgres:
 # **ACL**
 #
 acl_dict:
+  defaults:
+    inspect: |
+      WITH d AS (
+          SELECT
+              (aclexplode(datacl)).grantee AS grantee,
+              (aclexplode(datacl)).privilege_type AS priv
+          FROM pg_catalog.pg_database
+          WHERE datname = current_database()
+      )
+      SELECT NULL AS namespace, 'public' AS rolname
+      FROM d AS d0
+      LEFT OUTER JOIN d AS d1 ON d1.grantee = d0.grantee AND d1.priv <> d0.priv
+      WHERE d0.grantee = 0 AND d0.priv = 'TEMPORARY' AND d1.priv IS NULL
+      UNION
+      -- CONNECT ACL is not stored on template1 for PUBLIC. So fake it.
+      SELECT NULL AS namespace, 'public' AS rolname
+      WHERE current_database() = 'template1'
+    grant: |
+      REVOKE CONNECT ON DATABASE %(database)s FROM %(role)s
```

Comment:
Hi, yes it's odd. I should spell it the way around. Actually, I want to *revert* default ACL.